### DISCUSSION TITLE
SOP: Labels

### DISCUSSION THREAD

**Discussion by dan-menlo** [janhq/jan#3580] [How We Work]
## Goal

- Labels help us maintain strategic clarity vs. deluge of Issues, Bugs

## Current Setup

- `category: *` Area of app (e.g. app shell, engine management, hardware management, model running)
- `type: *` bug, epic, model request, engine request
- `p*`: p0, p1, p2
- `platform: *` OS-level issue
- `hardware: *` hardware-specific issues (e.g. Nvidia GPUs, Intel NPUs etc)

**Comment by ux-han** [comment]
I think this is repeating the same issue again which did not work the last time for us. Labels are over used for everything & hard to standardize. For example, ticket regarding Llama 3.1 request: `category: model running`, `type: model request`, `p*: p1`, `platform: window`, `platform: linux`, `platform: mac`, `hardware:...` _(listing all the hardwares)_?

All of this labels just for a ticket requesting model llama 3.1.
